.gitpod.yml

```diff
@@ -21,7 +22,9 @@ tasks:
     yarn download:sample-traces &&
     yarn
   command: >
-    yarn start:electron ../TraceCompassTutorialTraces/ --hostname=0.0.0.0 --ignore-gpu-blacklist --no-sandbox
+    export GITPOD_URL=$(gp url) &&
+    export TRACE_SERVER_URL=$(echo "$GITPOD_URL" | sed -E "s/(http[s]:\/\/)(.*)/\18080-\2\/tsp\/api/") &&
```

Comment:
Nicely done!

Reply:
`18080` - trace server port? Maybe parameterise this too?

Reply:
actually, gp url 8080 gives the url directly, I'll use that instead.